### ISSUE TITLE
Manage_Settings_DeveloperTool:dropdown is defined as null. (Accessibility)

### ISSUE THREAD

**Issue by nasc17** [microsoft/vscode#130708]
![](https://user-images.githubusercontent.com/69922333/128762467-6829c3cf-09cb-4ba6-bbf8-fe36d335d802.png)

Date: 2021-08-06T05:23:36.445Z
VS Code: 1.56.0
OS: Windows_NT x64 10.0.19043

Repro Steps:
Tab till settings button and press enter.
Tab till dropdown.
Open developer tool using ctrl+shift+I and check if the name is defined for dropdown.

Actual:
The name for the format:Identifier case dropdown is defined as null.

Expected:
The name should be defined as the label for the dropdown.

User Impact:
Screen Reader users will get wrong information regarding the combo box.

**Comment by roblourens** [comment]
I don't think anything is missing here. The setting container has information about the setting itself. When you focus the selectbox, it will read the value of the selectbox as expected.
